tests/TestClassFormModelBuilderDBForm.php

```diff
@@ -54,22 +57,26 @@ public function prepare()
 		$pContainerBuilder = new ContainerBuilder;
 		$pContainerBuilder->addDefinitions(ONOFFICE_DI_CONFIG_PATH);
 		$this->_pContainer = $pContainerBuilder->build();
+		$this->_pInstance = $this->getMockBuilder(FormModelBuilderDBForm::class)
```

Comment:
The test subject is a mock again. Only the test subjects' _dependencies_ are supposed to be a mocks, if needed (that means, if they have a side-effect that needs to be avoided). Please replace the MockBuilder with `new FormModelBuilderDBForm`.